cmd/snap/cmd_snap_op.go

```diff
@@ -383,35 +479,34 @@ func showDone(cli *client.Client, chg *client.Change, names []string, op string,
 		}
 		switch op {
 		case "install":
-			if needsPathWarning {
-				head := i18n.G("Warning:")
-				warn := fill(fmt.Sprintf(i18n.G("%s was not found in your $PATH. If you've not restarted your session since you installed snapd, try doing that. Please see https://forum.snapcraft.io/t/9469 for more details."), dirs.SnapBinariesDir), utf8.RuneCountInString(head)+1) // +1 for the space
-				fmt.Fprint(Stderr, esc.bold, head, esc.end, " ", warn, "\n\n")
-				needsPathWarning = false
-			}
+			if _, ok := notOnlyComps[snap.Name]; ok {
```

Comment:
the map values are bools, so this could just be:

if notOnlyComps[snap.Name] {

Reply:
Good catch, fixed now